src/components/filters/__tests__/input.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import { fireEvent, render } from "@testing-library/react"
+
+import InputFilter from "../input"
+
+describe("<InputFilter>", () => {
+  describe("renders correctly", () => {
+    it("with no values", () => {
+      const { container } = render(
+        <InputFilter
+          mode="numeric"
+          name="test"
+          placeholder="test"
+          apply={jest.fn()}
+        />
+      )
+
+      expect(container).toMatchSnapshot()
+    })
+
+    it("with autofocus after focusing", () => {
```

Comment:
I don't even know what it means 🙈

Reply:
It adds `autofocus` prop to the input after the first focus event IIRC. 
The reason is that changing the URL results in the page (re)load and we don't want to lose focus when typing.

Reply:
I see. no need to change existing code when porting. I think it's a good example of how we can make our tests better in the future ;)